### PR TITLE
3914: remove modal, add route and action 

### PR DESCRIPTION
**Pull request by sutt0n**
- removed edit meta modal
- added routing to new page view
- added action stub(ish?) to fill out form in state

### CODE REVIEW

File: web-client/src/views/DocketRecord/DocketRecord.jsx [hunk 1]
```diff
@@ -132,18 +127,11 @@ export const DocketRecord = connect(
...
-                          <Button
-                            link
-                            className="padding-0"
-                            icon="edit"
-                            onClick={() => {
-                              openEditDocketEntryMetaModalSequence({
-                                index: entry.index,
-                              });
-                            }}
+                          <a
```
Comment (kkoskelin) [inline web-client/src/views/DocketRecord/DocketRecord.jsx]: To ensure we get consistent styling, you might consider using the `<Button>` component but with an `href` -- which internally renders an anchor with with the provided `icon="edit"` with proper styling, etc.

File: web-client/src/presenter/actions/EditDocketRecordEntry/setDocketEntryMetaFormForEditAction.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+/**
+ * update state form with docket record entry and document properties
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.props the props passed to this action
+ * @param {object} providers.get the cerebral get function
+ * @param {object} providers.store the cerebral store
+ * @returns {void}
+ */
+export const setDocketEntryMetaFormForEditAction = ({ get, props, store }) => {
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/EditDocketRecordEntry/setDocketEntryMetaFormForEditAction.js:12]: are there tests pending for this new action?

File: web-client/src/router.js [hunk 3]
```diff
@@ -164,6 +164,19 @@ const router = {
       }),
     );
 
+    route(
+      '/case-detail/*/docket-entry/*/edit',
```
Comment (matthopson) [inline web-client/src/router.js]: I'm actually wondering if we should change this to `edit-meta` 🤔

We can decide later - I wouldn't hold this PR up for it

**Review by kkoskelin — APPROVED** [review]
Just a few observations that need not hold up this PR.

File: web-client/src/router.js (hunk 3 above)
Comment (sutt0n) [reply web-client/src/router.js]: ~I'll add a devex task for its consideration so we can groom/discuss it~ sounds good to me

File: web-client/src/presenter/actions/EditDocketRecordEntry/setDocketEntryMetaFormForEditAction.js (hunk 2 above)
Comment (sutt0n) [reply web-client/src/presenter/actions/EditDocketRecordEntry/setDocketEntryMetaFormForEditAction.js:12]: yes, they'll be added in another PR -- this PR was mainly focused on routing and removing modal

**Review by matthopson — APPROVED** [review]

**Pull request merged**